models/prices/prices_tokens.sql

```diff
@@ -85,7 +85,9 @@ VALUES
     ("comp-compoundd", "bnb", "COMP", "0x52ce071bd9b1c4b00a0b92d298c512478cad67e8", 18),
     ("cummies-cumrocket", "bnb", "CUMMIES", "0x27ae27110350b98d564b9a3eed31baebc82d878d", 18),
     ("dai-dai", "bnb", "DAI", "0x1af3f329e8be154074d8769d1ffa4ee058b1dbc3", 18),
-    ("dexe-dexe", "bnb", "DEXE", "0x039cb485212f996a9dbb85a9a75d898f94d38da6", 18),
+    ("dex-dexira", "bnb", "DEX", "0x147E07976E1ae78287c33aAFAab87760d32E50A5", 18),
+    ("dexe-dexe", "bnb", "DEXE", "0x039cb485212f996a9dbb85a9a75d898f94d38da6", 18),       
```

Comment:
this appears to be ethereum token: https://api.coinpaprika.com/v1/coins/dexe-dexe

Reply:
also, i believe all new entries should have contract address lowercase. maybe @aalan3 can help confirm

Reply:
This is correct @jeff-dude. In general all contract addresses (except for solana ones) in v2 need to be lowercase.